docker-compose.yml

```diff
@@ -53,8 +50,9 @@ services:
      - ./data/mongo:/data/db
      #- ./data/dump:/dump
     command: mongod --smallfiles
-    ports:
-      - 27017:27017
+    # Don't expose unprotected mongo db to the world, needs to be accessible only from within docker network
```

Comment:
What do you think about change the comment here for something short that explains for the user that by uncomment these lines, is possible to make a temporary access outside docker?

I'm saying this because these changes have to be documented somewhere else, but a simple inline comment could be sufficient.

Maybe something like

```sh
    # Remove the commends from next lines to expose ElasticSearch port to host network, local and remote access (not recommended!)
    # ports:
    #   - 9200:9200

(...)

    # Remove the commends from next lines to expose MongoDB port to host network, local and remote access (not recommended!)
    # ports:
    #   - 27017:27017
```

The text of the description is just a draft. But the main difference is that it does not explain why the line is commented, but one reason someone new to docker _could_ remove the comment but at same time do not recommend it.

Reply:
I still don't understand how these pull requests work, specifically how to merge new upstream changes before doing any changes myself. Sorry for the confusion.